web/modules/custom/solution/solution.module

```diff
@@ -260,8 +264,13 @@ function solution_rdf_entity_view(array &$build, EntityInterface $entity, Entity
     $build['downloads'] = [
       '#type' => 'container',
       '#stats_topic' => t('Downloads'),
-      0 => ['#markup' => (string) $total],
     ];
+    // Only add #markup if we want to show it.
+    if ($total >= 1) {
+      $build['downloads'][0]['#markup'] = (string) $total;
```

Comment:
We can omit 0 and do `$build['downloads'][] = ['#markup' => $total]`. Also I don't see why we need the string casting.

Reply:
True! Actually I also don't understand the string casting just it was there.. Will remove it!

Reply:
Removed!